hooks-admin.php

```diff
@@ -199,7 +199,6 @@
 	add_filter( 'attachment_fields_to_edit', '\Pressbooks\Admin\Attachments\add_metadata_attachment', 10, 2 );
 	add_filter( 'attachment_fields_to_save', '\Pressbooks\Admin\Attachments\save_metadata_attachment', 10, 2 );
 	add_filter( 'edit_form_top', [ '\Pressbooks\Book', 'notifyBisacCodesRemoved' ] );
-	add_filter( 'wp_handle_upload_prefilter', '\Pressbooks\Admin\Metaboxes\validate_contributor_picture_size', 10, 1 );
```

Comment:
It is validated using wp media here: https://github.com/pressbooks/pressbooks/blob/eeeaa48b18ee796517bf6349fa73a20405dbc789/assets/src/scripts/contributors.js#L172